elastic-training/controllers/elastichorovodjob_controller.go

```diff
@@ -288,12 +402,14 @@ func (r *ElasticHorovodJobReconciler) desiredLauncherJob(ehjob aiv1alpha1.Elasti
 	scriptMode := int32(0744)
 	one := int32(1)
 	sshSetupCommand := "mkdir -p /root/.ssh; cp /etc/secrets/* /root/.ssh/; chmod 644 /root/.ssh/authorized_keys;"
-	scriptSetupCmd := fmt.Sprintf("mkdir -p /scripts; cp /etc/scripts/* /scripts/; sed -i 's/SERVICENAME/%s/' /scripts/discover_hosts.sh;", svcName)
+	scriptSetupCmd := fmt.Sprintf("mkdir -p /scripts; mkdir -p /elastic_scripts; cp /etc/scripts/* /scripts/; sed -i 's/SERVICENAME/%s/' /scripts/discover_hosts.sh;", svcName)
```

Comment:
I don't see this /elastic_scripts used anywhere?

Reply:
Removed